share/doc/homebrew/Acceptable-Formulae.md

```diff
@@ -104,12 +105,9 @@ running a package manager.
 
 ### Stuff that builds a .app
```

Comment:
I don't care either way, but it's inconsistent with the text two or three lines below.